### ISSUE TITLE
Blend is internal. Harmonize, a necessary function for custom colors, is unavailable as a result

### ISSUE THREAD

**Issue by Number869** [jordond/MaterialKolor#90]
the usefulness of harmonization is showcased at https://m3.material.io/styles/color/advanced/adjust-existing-colors#1cc12e43-237b-45b9-8fe0-9a3549c1f61e. a harmonized tonal palette could hypothetically look like 
```kotlin
val desiredColor = if (harmonize) harmonize(MaterialTheme.colorScheme.primary.toArgb(), someColor) else someColor

val palette = TonalPalette.fromInt(desiredColor)

**Comment by Number869** [comment]
i forgot one thing. a tool to adjust a color's saturation to match another color's saturation is also kinda needed. a palette made from a saturated color theme will look off when used on a dull theme

**Comment by Number869** [comment]
Update: i have made a draft implementation and an couple examples for it. https://github.com/jordond/MaterialKolor/commit/29dfe9485b76f4051a3578e053e4b585a878f400

**Comment by jordond** [comment]
I've whipped up a PR with the blend functionality, and with your second suggestion.

**Comment by jordond** [comment]
It's available now in `1.4.0-alpha05`
